Source/Evolve/WarpXEvolve.cpp

```diff
@@ -255,7 +253,7 @@ WarpX::Evolve (int numsteps)
         amrex::Print()<< "STEP " << step+1 << " ends." << " TIME = " << cur_time
                       << " DT = " << dt[0] << "\n";
         Real walltime_end_step = amrex::second();
-        walltime = walltime_end_step - walltime_start;
+        walltime += walltime_end_step - walltime_beg_step;
         amrex::Print()<< "Walltime = " << walltime
```

Comment:
I would also rename "walltime" - walltime is generally associated with the overall job runtime. But what we measure here does not include initialization.

Suggested name: "Evolve time =" or so

Reply:
I changed walltime to evolve_time and the amrex::Print() now has "Evolve time = "